server/impl/src/main/java/com/walmartlabs/concord/server/process/pipelines/processors/ConfigurationProcessor.java

```diff
@@ -90,6 +87,10 @@ public Payload process(Chain chain, Payload payload) {
         List<String> activeProfiles = ProcessConfigurationUtils.getActiveProfiles(payloadCfg, attachedCfg, workspaceCfg, projectCfg, orgCfg);
         payload = payload.putHeader(Payload.ACTIVE_PROFILES, activeProfiles);
 
+        // consolidate out variables
+        Set<String> outVars = new HashSet<>(ProcessConfigurationUtils.getOutVars(payloadCfg, attachedCfg, workspaceCfg));
```

Comment:
I might be reading it wrong, but it looks like `getOutVars(...)` returns the first occurrence of `out` rather than a merged set of all `out` values?